Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.Data.cs

```diff
@@ -19,29 +19,29 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel
     /// </summary>
     public partial class WrapPanel
     {
-        internal class UvMeasure
+        [System.Diagnostics.DebuggerDisplay("U = {U} V = {V}")]
+        internal struct UvMeasure
         {
-            internal double U { get; set; }
+            internal const double FACTOR = 10000;
 
-            internal double V { get; set; }
+            private int u, v;
+            internal double U { get { return u / FACTOR; } set { u = (int)(value * FACTOR); } }
```

Comment:
I think this getter/setter could be replaced with Math.Floor ? I didn't try my suggestion but it seems you convert to integer then divide to get rid of decimal point to get the floor of the number